### PR TITLE
Remove ttl && tensorrt headers + fix squash + fix images

### PR DESCRIPTION
**Pull request by ganler**

### CODE REVIEW

**Comment by lgarithm** [comment]
How is **squash** been fixed in this PR?

**Comment by ganler** [comment]
> How is **squash** been fixed in this PR?

I rebased the commits which failed in CI.

**Comment by lgarithm** [comment]
Will this be force pushed to master?

**Comment by ganler** [comment]
> Will this be force pushed to master?

I did force pushed this branch after using `git rebase`.

**Comment by ganler** [comment]
@lgarithm Do you have some time to look at this?

File: cmake/user_codes.cmake [hunk 1]
```diff
@@ -0,0 +1,13 @@
+INCLUDE(${CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
+
+FILE(GLOB_RECURSE USER_CODES ${CMAKE_SOURCE_DIR}/examples/user_codes/*.cpp)
+
+foreach(USERCODE_FULL_PATH ${USER_CODES})
+    MESSAGE(STATUS ">>> [USER CODES] TO BUILD ${USERCODE_FULL_PATH}")
+    GET_FILENAME_COMPONENT(USER_CODE_NAME ${USERCODE_FULL_PATH} NAME_WE)
+
+    SET(USER_CODA_TAR user.${USER_CODE_NAME})
```
Comment (lgarithm) [inline cmake/user_codes.cmake:9]: USER_CODA_TAR -> USER_CODE_TAR

File: include/hyperpose/utility/data.hpp [hunk 2]
```diff
@@ -18,9 +19,11 @@ struct feature_map_t {
     /// Constructor.
     /// \param name Tensor name. (Often from DNN engine graphs)
     /// \param tensor Tensor data.
-    inline feature_map_t(std::string name, ttl::tensor<float, 3> tensor)
+    /// \param shape Shape of tensor. (no batch dimension)
+    inline feature_map_t(std::string name, std::unique_ptr<char[]>&& tensor, std::vector<int> shape)
```
Comment (lgarithm) [inline include/hyperpose/utility/data.hpp:23]: use `std::array<int, 3>` for shape

Comment (ganler) [reply include/hyperpose/utility/data.hpp:23]: > use `std::array<int, 3>` for shape

I made this change cause I am implementing pose proposal network's post processing, where has different dim requirement.

One of the output tensor has a dim of [17, 9, 9, 12, 12].

**Pull request merged**